diff --git a/google/cloud/vision/v1/geometry.proto b/google/cloud/vision/v1/geometry.proto
index 30aa355c7..0394bb54d 100644
--- a/google/cloud/vision/v1/geometry.proto
+++ b/google/cloud/vision/v1/geometry.proto
@@ -17,7 +17,7 @@ syntax = "proto3";
 package google.cloud.vision.v1;
 
 option cc_enable_arenas = true;
-option go_package = "cloud.google.com/go/vision/apiv1/visionpb;visionpb";
+option go_package = "cloud.google.com/go/vision/v2/apiv1/visionpb;visionpb";
 option java_multiple_files = true;
 option java_outer_classname = "GeometryProto";
 option java_package = "com.google.cloud.vision.v1";
diff --git a/google/cloud/vision/v1/image_annotator.proto b/google/cloud/vision/v1/image_annotator.proto
index bb022ffc9..46953e684 100644
--- a/google/cloud/vision/v1/image_annotator.proto
+++ b/google/cloud/vision/v1/image_annotator.proto
@@ -30,7 +30,7 @@ import "google/type/color.proto";
 import "google/type/latlng.proto";
 
 option cc_enable_arenas = true;
-option go_package = "cloud.google.com/go/vision/apiv1/visionpb;visionpb";
+option go_package = "cloud.google.com/go/vision/v2/apiv1/visionpb;visionpb";
 option java_multiple_files = true;
 option java_outer_classname = "ImageAnnotatorProto";
 option java_package = "com.google.cloud.vision.v1";
diff --git a/google/cloud/vision/v1/product_search.proto b/google/cloud/vision/v1/product_search.proto
index b68f3fdc8..98ece5457 100644
--- a/google/cloud/vision/v1/product_search.proto
+++ b/google/cloud/vision/v1/product_search.proto
@@ -22,7 +22,7 @@ import "google/cloud/vision/v1/product_search_service.proto";
 import "google/protobuf/timestamp.proto";
 
 option cc_enable_arenas = true;
-option go_package = "cloud.google.com/go/vision/apiv1/visionpb;visionpb";
+option go_package = "cloud.google.com/go/vision/v2/apiv1/visionpb;visionpb";
 option java_multiple_files = true;
 option java_outer_classname = "ProductSearchProto";
 option java_package = "com.google.cloud.vision.v1";
diff --git a/google/cloud/vision/v1/product_search_service.proto b/google/cloud/vision/v1/product_search_service.proto
index 02e367b0c..f97b443ca 100644
--- a/google/cloud/vision/v1/product_search_service.proto
+++ b/google/cloud/vision/v1/product_search_service.proto
@@ -28,7 +28,7 @@ import "google/protobuf/timestamp.proto";
 import "google/rpc/status.proto";
 
 option cc_enable_arenas = true;
-option go_package = "cloud.google.com/go/vision/apiv1/visionpb;visionpb";
+option go_package = "cloud.google.com/go/vision/v2/apiv1/visionpb;visionpb";
 option java_multiple_files = true;
 option java_outer_classname = "ProductSearchServiceProto";
 option java_package = "com.google.cloud.vision.v1";
diff --git a/google/cloud/vision/v1/text_annotation.proto b/google/cloud/vision/v1/text_annotation.proto
index 303186088..e84a596a2 100644
--- a/google/cloud/vision/v1/text_annotation.proto
+++ b/google/cloud/vision/v1/text_annotation.proto
@@ -19,7 +19,7 @@ package google.cloud.vision.v1;
 import "google/cloud/vision/v1/geometry.proto";
 
 option cc_enable_arenas = true;
-option go_package = "cloud.google.com/go/vision/apiv1/visionpb;visionpb";
+option go_package = "cloud.google.com/go/vision/v2/apiv1/visionpb;visionpb";
 option java_multiple_files = true;
 option java_outer_classname = "TextAnnotationProto";
 option java_package = "com.google.cloud.vision.v1";
diff --git a/google/cloud/vision/v1/web_detection.proto b/google/cloud/vision/v1/web_detection.proto
index 36d36b8b8..5f36bb7ac 100644
--- a/google/cloud/vision/v1/web_detection.proto
+++ b/google/cloud/vision/v1/web_detection.proto
@@ -17,7 +17,7 @@ syntax = "proto3";
 package google.cloud.vision.v1;
 
 option cc_enable_arenas = true;
-option go_package = "cloud.google.com/go/vision/apiv1/visionpb;visionpb";
+option go_package = "cloud.google.com/go/vision/v2/apiv1/visionpb;visionpb";
 option java_multiple_files = true;
 option java_outer_classname = "WebDetectionProto";
 option java_package = "com.google.cloud.vision.v1";
